admin_manual/configuration_files/default_files_configuration.rst

```diff
@@ -18,3 +18,11 @@ They appear on the user's ownCloud Files page just like any other files.
 
 .. image:: ../images/skeleton-files1.png
 
+Additional Configuration
+------------------------
+
+The config option ``skeletondirectory`` available in your
```

Comment:
Just shameless copied from https://github.com/owncloud/documentation/commit/781551f443d67b5d7087e3aba726f909026f8650 :-p so will also fixed this there.